apisix/plugins/openwhisk.lua

```diff
@@ -104,8 +106,35 @@ function _M.access(conf, ctx)
         return 503
     end
 
+    -- parse OpenWhisk JSON response
+    -- OpenWhisk supports two types of responses, the user can return only
+    -- the response body, or set the status code and header.
+    local result, err = core.json.decode(res.body)
+
+    if err then
```

Comment:
```suggestion
    if not result then
```
would be more common

Reply:
fixed.